winton_kafka_streams/processor/serialization/avro.py

```diff
@@ -0,0 +1,68 @@
+from confluent_kafka.avro import CachedSchemaRegistryClient, MessageSerializer
+from confluent_kafka.avro import loads as avro_loads
+
+from ._serde import extract_config_property
+from ._deserializer import Deserializer
+from ._serializer import Serializer
+
+
+class AvroHelper:
```

Comment:
I think a better name would be `AvroSerde` as it's not so much a helper as the whole implementation.

Can I take it the reason to have both serialisation and deserialisation centralised here is because configuration is common?

Reply:
I've now seen there's a `AvroSerde` below. Perhaps `AvroSerdeImpl` instead of `AvroHelper` then.